_install-and-configure/install-opensearch/debian.md

```diff
@@ -0,0 +1,470 @@
+   ```
+1. Once complete, you can run OpenSearch.
+    ```bash
+    sudo systemctl enable --now opensearch
+    ```
+1. Verify that OpenSearch launched correctly.
+    ```bash
+    sudo systemctl status opensearch
+    ```
+
+## Step 2: Configure important system settings
+
+Before launching OpenSearch you should review some [important system settings]({{site.url}}{{site.baseurl}}/opensearch/install/important-settings/){:target='\_blank'}.
+1. Disable memory paging and swapping performance on the host to improve performance.
+   ```bash
+   sudo swapoff -a
+   ```
+1. Increase the number of memory maps available to OpenSearch.
+   ```bash
+   # Edit the sysctl config file
+   sudo vi /etc/sysctl.conf
+
+   # Add a line to define the desired value
+   # or change the value if the key exists,
+   # and then save your changes.
+   vm.max_map_count=262144
+
+   # Reload the kernel parameters using sysctl
+   sudo sysctl -p
+
+   # Verify that the change was applied by checking the value
+   cat /proc/sys/vm/max_map_count
+   ```
+
+## Step 3: (Optional) Test OpenSearch
+
+Before proceeding with any configuration, you should test your installation of OpenSearch. Otherwise, it can be difficult to determine whether future problems are due to installation issues or custom settings you applied after installation.
+
+When OpenSearch is installed using the Debian package, some demo security settings are automatically applied. This includes self-signed TLS certificates and several users and roles. If you would like to configure these yourself, see [Set up OpenSearch in your environment](#step-4-set-up-opensearch-in-your-environment).
+
+An OpenSearch node in its default configuration (with demo certificates and users with default passwords) is not suitable for a production environment. If you plan to use the node in a production environment, you should, at a minimum, replace the demo TLS certificates with your own TLS certificates and [update the list of internal users and passwords]({{site.url}}{{site.baseurl}}/security-plugin/configuration/yaml). See [Security configuration]({{site.url}}{{site.baseurl}}/security-plugin/configuration/index/) for additional guidance to ensure that your nodes are configured according to your security requirements.
+{: .warning}
+
+1. Send requests to the server to verify that OpenSearch is running. Note the use of the `--insecure` flag, which is required because the TLS certificates are self-signed.
+   - Send a request to port 9200:
+      ```bash
+      curl -X GET https://localhost:9200 -u 'admin:admin' --insecure
+      ```
+      You should get a response that looks like this:
+      ```bash
+      {
+         "name" : "hostname",
+         "cluster_name" : "opensearch",
+         "cluster_uuid" : "oG0pPKNWT42HT3UqWgmxuw",
+         "version" : {
+            "distribution" : "opensearch",
+            "number" : "2.4.0",
+            "build_type" : "deb",
```

Comment:
Requested full response bodies from 2.5 version since I don't have it available to me and manually changing fields will invalidate the other info like build hash and date.

Reply:
I understand now - I was previously under the impression that all 2.x versions will have Debian distribution but only 2.5+ will have it.